build.js

```diff
@@ -548,7 +560,7 @@ function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
       .append($('<td></td>')
         .attr('id', getHtmlId(id))
         .append('<span><a class="anchor" href="#' + getHtmlId(id) + '">&sect;</a>' + name + footnoteHTML(t) + '</span></td>')
-        .append(testScript(t.exec, compiler, rowNum++))
+        .append(testScript(eval(t.exec), compiler, rowNum++))
```

Comment:
we reallllly shouldn't use `eval`. why is this needed?

Reply:
Because ```testScript``` expects a function.

Reply:
That’s why the function should be stored in a .js file - functions aren’t data, and json is for data.

Reply:
It now stores the tests in JS and the results in JSON.